user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -306,31 +312,31 @@ def train(self,
                                                           n_trials=n_trials,
                                                           base_model=base_model,
                                                           features_csv_dir=features_csv_dir,
-                                                          config=config)
+                                                          qualx_config=qualx_config)
 
         tool_obj = Train(platform_type=train_args['runtimePlatform'],
                          output_folder=output_folder,
                          wrapper_options=train_args)
         tool_obj.launch()
 
-    def train_and_evaluate(self, config: str = None):
+    def train_and_evaluate(self, qualx_pipeline_config: str = None):
         """The Train and Evaluate cmd trains an XGBoost model and evaluates it using matched CPU and GPU eventlogs.
 
         This API supports online training by allowing multiple invocations with new data.
         Each invocation will create a new dataset JSON file with an incrementing number.
 
-        :param config: Path to YAML config file containing the required training parameters.
+        :param qualx_pipeline_config: Path to YAML config file containing the required training parameters.
```

Comment:
@leewyang does this fallback to `resources/qualx-conf.yaml` if not provided by user?